cloudbuild.yaml

```diff
@@ -14,10 +14,118 @@
 
 
 steps:
```

Comment:
In general, I think it would be a good idea to add time outs to these steps. So that way if something goes wrong, it doesn't hang forever and eat up resources.

Reply:
Do you know what typical timeout values are? For building the docker image, maybe 1 minute is reasonable? For the Terraform steps, I'm concerned about adding a timeout since they depend on how many environments are located in the environments folder, and the configs are applied to each one in the current build config.